### ISSUE TITLE
Host bots

### ISSUE THREAD

**Issue by UncleVasya** [UncleVasya/Dota2-EU-Ladder#9]
Dota bot:
- [x] create dota lobby;
- [x] get ids of players in slots;
- [x] !balance command:
    - [x] check if all players are present in DB;
    - [x] balance teams, write result in lobby chat;
- [x] !start command to launch lobby;
   - [x] check that teams are made according to balancer output;
 -----------
Discord bot:
- [ ] paste balance result in chat after dota bot balanced teams;
- [ ] !lobby command to show current lobby info;
- [ ] !invite command to invite user to current lobby;
- [ ] react with emote on lobby info to get invited (thanks sbx320 for the idea);
-----------
Hosting:
- [ ] setup bots on openshift;
-----------
Future:
- [ ] bind several dota accounts to our player account;
- [ ] if unresgistered player joins lobby and has his MMR shown, we can add him to DB right away;
- [ ] managing several lobbies with discord bot:
    - [ ] !invite 1, !invite 2 etc;
- [x] !balance 3 to show and set specific balance answer;
- [ ] after !balance command:
    - [ ] bot kicks players from wrong slots;
    - [ ] don't allow players to join slots on the wrong team;

**Comment by UncleVasya** [comment]
Approach without lobby commands:

- [ ] balance as soon as 10 players join slots;
- [ ] start as soon as people joined correct slots;
- [ ] if balanced players leave lobby or join broadcaster slots, reset balance;

**Comment by UncleVasya** [comment]
Looks like grabing match result is perfectly possible for host bot:
https://github.com/Arcana/node-dota2/issues/240

**Comment by UncleVasya** [comment]
depends on: https://github.com/UncleVasya/dota2/issues/1

**Comment by UncleVasya** [comment]
Commands users ask for:
- [ ] !teamkick;
- [ ] !forcestart;

**Comment by UncleVasya** [comment]
!teamkick and !forcestart will be done in #14

**Comment by UncleVasya** [comment]
Discord bot moved to separate issue: #15
